tests/contract_testcases/evm/call/call.sol

```diff
@@ -4,4 +4,6 @@
                 (bool s, bytes memory bs) = a.delegatecall{value: 2}("");
             }
         }
-        
\ No newline at end of file
+        
+// ----
+// error (117-145): 'delegatecall' cannot have value specifed
```

Comment:
What if we write the line numbers inside the parenthesis? This way, the test is even more human readable.

Reply:
I agree and updated the tests accordingly